supabase/pending/billing_historicals.sql

```diff
@@ -0,0 +1,234 @@
+create table "public"."billing_historicals" (
```

Comment:
we're not doing `pending` migrations anymore. Instead, add append-only new migrations to `supabase/migrations`.

Reply:
Okay, so this means that I'll largely be duplicating `billing_report_202308` in the new migration